rust_dev_preview/cross_service/rest_ses/src/work_item/work_item_archived.rs

```diff
@@ -0,0 +1,129 @@
+//! WorkItemArchived represents several ways to specify the archival status of a WorkItem.
+
+use serde::{de, Deserialize, Serialize};
+use std::convert::TryFrom;
+
+use crate::work_item::WorkItemError;
+
+/// An enum to represent the archival status of a WorkItem.
+/// This field has varying representations at different parts of the stack, so needs some specialized serde visitors.
```

Comment:
/// Because this field has varying representations at different parts of the stack, it needs some specialized serde visitors.